stumpy/mstump.py

```diff
@@ -481,9 +562,18 @@ def _multi_distance_profile(
     return D
 
 
-@core.non_normalized(maamp_multi_distance_profile)
+@core.non_normalized(
+    maamp_multi_distance_profile, exclude=["T_subseq_isconstant", "normalize"]
```

Comment:
Please flip the order to be `["normalize", "T_subseq_isconstant"]` to reflect the order in which they appear in the function signature